src/llmcompressor/entrypoints/oneshot.py

```diff
@@ -123,6 +126,13 @@ def __init__(
         # initialize the model and processor
         pre_process(model_args)
 
+        # offload to cpu if possible
+        if "cuda" in str(model_args.oneshot_device) and torch.cuda.is_available():
+            remove_hook_from_module(model_args.model, recurse=True)
```

Comment:
What hooks would we be removing at this point?

Reply:
Theoretically none, but this is to foolproof against any previous dispatches
FYI, the remove_hook_from_module has been moved to the `offloaded_dispatch`
https://github.com/neuralmagic/compressed-tensors/pull/357